NCSDK-3416: fix gs assistant installation information (#1361)
NordicPlayground/fw-nrfconnect-nrf: doc/nrf/gs_assistant.rst

The application allows you to check which of the required tools are already installed on your computer and verify the installation.
It also guides you through installing the parts of the toolchain that you are missing.
In addition, it will help you clone the |NCS| repositories and set up your first project in |SES|.

.. _gs_assistant_installing-gsa:

Installing the |GSA|
********************

The |GSA| app is available in `nRF Connect for Desktop`_, a cross-platform tool available for Windows, Linux, and macOS.

To install the Assistant app, complete the following steps:

.. _assistant_setup:

1. `Download nRF Connect for Desktop`_ for your operating system.
#. Install and run the tool on your machine.
#. In the APPS section, click :guilabel:`Install` next to |GSA|.

.. figure:: images/gs-assistant_installation.PNG
   :alt: The nRF Connect for Desktop main window

   The nRF Connect for Desktop main window

The app is installed on your machine, and the :guilabel:`Install` button changes to :guilabel:`Open`.

.. _gs_assistant_installing-ncs:

Installing the |NCS|
********************

> Once you installed |GSA|, open it in nRF Connect for Desktop.

Reviewer: ```suggestion
Once you have installed |GSA|, open it in nRF Connect for Desktop.
```

> The |GSA| app will open with the Overview section displayed in the main window.

Reviewer: ```suggestion
The |GSA| app opens with the Overview section displayed in the main window.
```

You can now go through each of the sections and complete the listed installation steps.

To provide installation variables, use the Variables section.
If you need to edit any of the fields, click on it.

.. figure:: images/gs-assistant_variables.PNG
   :alt: The Variables section of the |GSA| app main window, cropped

   The Variables section

.. _gs_assistant_installing-ncs_verify:

Verification options
====================

> Next to each step you can use one or both of the following buttons:

Reviewer: ```suggestion
Next to each step, you can use one or both of the following buttons:
```

.. _gs_assistant:

> .. |GSA| replace:: Getting Started Assistant

Reviewer 2: This should go into the shortcuts.txt file.

> .. gs_assistant_intro

Reviewer 2: Not needed (since you have _gs_assistant at the top of the file).

> Installing the |NCS| with |GSA|

Reviewer 2: ```suggestion
Installing the |NCS| with the |GSA|
```

###############################

The |GSA| application provides you with step-by-step instructions for installing the toolchain for |NCS|.

The application allows you to check which of the required tools are already installed on your computer and verify the installation.
It also guides you through installing the parts of the toolchain that you are missing.
In addition, it will help you clone the |NCS| repositories and set up your first project in |SES|.

> .. gs_assistant_installing-gsa

Reviewer 2: All of these need a : at the end and a _ at the beginning (or they will just be comments, but not IDs that can be linked to).

Installing the |GSA|
********************

The |GSA| app is available in `nRF Connect for Desktop`_, a cross-platform tool available for Windows, Linux, and macOS.

To install the Assistant app, complete the following steps:

.. _assistant_setup:

1. `Download nRF Connect for Desktop`_ for your operating system.
#. Install and run the tool on your machine.
#. In the APPS section, click :guilabel:`Install` next to |GSA|.

.. figure:: images/gs-assistant_installation.PNG
   :alt: The nRF Connect for Desktop main window
   
   The nRF Connect for Desktop main window

The app is installed on your machine, and the :guilabel:`Install` button changes to :guilabel:`Open`.

.. gs_assistant_installing-ncs

Installing the |NCS|
********************

Once you installed |GSA|, open it in nRF Connect for Desktop.
The |GSA| app will open with the Overview section displayed in the main window.

> .. figure:: images/gs-assistant_overview.PNG

Reviewer 2: I don't think this screenshot is needed (people will see it when they open the app).

   :alt: The |GSA| app main window
   
   The |GSA| app main window

You can now go through each of the sections under Overview and complete the listed installation steps.

To provide installation variables, use the Variables section.
If you need to edit any of the fields, click on it.

.. figure:: images/gs-assistant_variables.PNG
   :alt: The Variables section of the |GSA| app main window, cropped
   
   The Variables section

.. gs_assistant_installing-ncs_verify

Verification options
====================

Next to each step you can use one or both of the following buttons:

- :guilabel:`Mark done` -- marks a step as completed.
- :guilabel:`Verify` -- checks if the installation is correct.

The verification process is reflected in the vertical progress bar animation.
Once the verification is successful, the vertical progress bar changes color to green and the Log section is updated.
At the same time, the :guilabel:`Mark done` button changes to :guilabel:`Mark not done`.

> .. figure:: images/gs-assistant_UI.gif

Reviewer 2: nice :)
